outboundSender.go

```diff
@@ -448,13 +451,29 @@ func (obs *CaduceusOutboundSender) isValidTimeWindow(now, dropUntil, deliverUnti
 	return true
 }
 
+func (obs *CaduceusOutboundSender) Empty() {
+	for obs.queueEmpty == false {
```

Comment:
```suggestion
	for !obs.queueEmpty {
```
this is a superficial thing, feel free to ignore.